### PR TITLE
Update blog posts list page

### PR DESCRIPTION
**Pull request by Dhruvi16**
## Description

This updates blog posts page.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: themes/vocabulary_theme/templates/macros/posts.html [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+          {% endif %}
+          </figure>
+        </header>
+        <div class="blog-content">
+          <h4 class="b-header"><a class="blog-title" href="{{ post|url }}">{{ post.title }}</a></h4>
+          <span class="blog-author">by <a href="{{ author|url }}">{{ render_author_name(author) }}</a>
+          on {{ post.pub_date|dateformat("YYYY-MM-dd") }}</span>
+          <!-- TODO : Discuss how to extract an excerpt -->
+          <div class="excerpt">
+            {{ post.body | excerpt | string | striptags() | truncate(100) }}
```
Comment (hugosolar) [inline themes/vocabulary_theme/templates/macros/posts.html:24]: nice!

Comment (Dhruvi16) [reply themes/vocabulary_theme/templates/macros/posts.html:24]: I wanted to suggest that we can update `How to write a blog` with adding guidelines about adding an excerpt in the start of the blog.

Comment (hugosolar) [reply themes/vocabulary_theme/templates/macros/posts.html:23]: Go for it 😄

**Comment by Dhruvi16** [comment]
> On desktop looks fine but I'm still getting padding issues on the phone screen
> 
> Screen Shot 2020-07-06 at 3 44 18 PM

I will reduce the padding here then but this PR refers to the changes on this path - `/blog/entries/`

**Comment by Dhruvi16** [comment]
And as far as the desktop version is concerned the design has a lot of padding( [here](https://www.figma.com/file/mttcugI1UxvCJRKE5sdpbO/Mockups?node-id=885%3A1561) ) so I had to but I will make sure that this padding does not affect the mobile version.

**Comment by hugosolar** [comment]
> > On desktop looks fine but I'm still getting padding issues on the phone screen
> > Screen Shot 2020-07-06 at 3 44 18 PM
> 
> I will reduce the padding here then but this PR refers to the changes on this path - `/blog/entries/`

ohh you're right! sorry.. my bad

**Pull request merged**